src/pages/TrueSightV2/components/FeedbackSurvey.tsx

```diff
@@ -0,0 +1,178 @@
+import { Trans } from '@lingui/macro'
+import { useEffect, useMemo, useState } from 'react'
+import { X } from 'react-feather'
+import { Text } from 'rebass'
+import styled, { css } from 'styled-components'
+
+import { ButtonOutlined, ButtonPrimary } from 'components/Button'
+import Modal from 'components/Modal'
+import { RowBetween, RowFit } from 'components/Row'
+import useTheme from 'hooks/useTheme'
+import { useGetParticipantKyberAIInfo } from 'state/user/hooks'
+
+import { ParticipantStatus } from '../types'
+
+const Wrapper = styled.div`
+  width: min(95vw, 480px);
+  border-radius: 20px;
+  padding: 20px;
+  display: flex;
+  gap: 20px;
+  flex-direction: column;
+  b {
+    color: ${({ theme }) => theme.text};
+  }
+`
+
+const WidgetWrapper = styled.div<{ show?: boolean }>`
+  position: fixed;
+  right: 0;
+  top: 310px;
+  transition: filter 0.1s ease-out, right 1s ease-out, visibility 1s;
+  z-index: 20;
+  transform-origin: 100% 100%;
+  transform: rotate(-90deg);
+  border-radius: 8px 8px 0px 0px;
+  background: ${({ theme }) => theme.primary + '32'};
+  padding: 4px 8px;
+  color: ${({ theme }) => theme.primary};
+  cursor: pointer;
+  ${({ show }) =>
+    !show &&
+    css`
+      right: -25px;
+      visibility: hidden;
+    `};
+
+  :hover {
+    filter: brightness(1.2);
+  }
+`
+
+const XWrapper = styled.span`
+  padding: 2px;
+  border-radius: 50%;
+  transition: all 0.1s;
+  svg {
+    display: block;
+  }
+  :hover {
+    background-color: ${({ theme }) => theme.primary + '36'};
+  }
+`
+
+const LOCALSTORAGE_MODAL_SHOWED = 'showedKyberAIFeedbackSurvey'
+const LOCALSTORAGE_WIDGET_SHOWED = 'showedKyberAIFeedbackSurveyWidget'
+const MOMENT_THIS_SURVEY_RELEASE = 1689768000
+export default function FeedbackSurvey() {
+  const [isOpen, setIsOpen] = useState(false)
+  const [isOpenWidget, setIsOpenWidget] = useState(false)
+  const theme = useTheme()
+  const { updatedAt, status } = useGetParticipantKyberAIInfo()
+
+  const isValid = useMemo(
+    () => updatedAt < MOMENT_THIS_SURVEY_RELEASE && status === ParticipantStatus.WHITELISTED,
```

Comment:
`updatedAt < MOMENT_THIS_SURVEY_RELEASE`
This means we only show this popup for old whitelisted users, right?

Reply:
correct